src/objects/Hierarchy_Settings__c.object

```diff
@@ -75,7 +75,7 @@
         <description>The namespace of the Gift Processing package.</description>
```

Comment:
Minor change, but should we also update the description to say "... Gift Entry Management package"? 

I do actually notice some references to "Gift Processing" in the Docs, so maybe this isn't a big deal (actually, I KNOW it isn't a big deal...)

Reply:
Do we want to go with Gift Entry Management package, or Gift Entry Management application?  I don't mind changing it, either way.

Reply:
It's a mouthful, but I'd vote for "Gift Entry Management Managed Package", or "GEM Managed Package" because the value of this setting has to match the Namespace of the GEM package that is installed in the org.